sources/packages/web/src/services/StudentService.ts

```diff
@@ -74,9 +73,10 @@ export class StudentService {
       await ApiClient.Students.synchronizeFromUserToken();
       return true;
     } catch (error: unknown) {
+      // TODO: Get MISSING_STUDENT_ACCOUNT /services/http/StudentApi.ts, It was remove for now as it was throwing "Super expression must either be null or a function, not undefined" error
       if (
         error instanceof ApiProcessError &&
-        error.errorType === MISSING_STUDENT_ACCOUNT
+        error.errorType === "MISSING_STUDENT_ACCOUNT"
```

Comment:
Any particular reason this was changed ?

Reply:
ya, Henry, remember yesterday in the dev I chat I posted my local loading after merge conflicts. `MISSING_STUDENT_ACCOUNT` import was the issue, I think. if I comment on the `MISSING_STUDENT_ACCOUNT` it was working. so, for the sake of PR, I have removed the import and added a TODO in line 76.  below is the error it was throwing, `"Super expression must either be null or a function, not undefined" `. let me know, if you guys know or face similar error